etc/disable-common.inc

```diff
@@ -172,3 +172,29 @@ blacklist ${PATH}/roxterm-config
 blacklist ${PATH}/terminix
 blacklist ${PATH}/urxvtc
 blacklist ${PATH}/urxvtcd
+
+# disable common suid programms
+blacklist ${PATH}/firejail
```

Comment:
If I'm reading this correctly, you comment out several lines such as `# blacklist ${PATH}/su` around lines 148 but then re-list them around line 180. Assuming that it just isn't a mistake on my part, why is that?